http_handlers.go

```diff
@@ -515,9 +516,14 @@ func handlePacks(c echo.Context) error {
 	)
 
 	if langCode != "" {
-		pack, err := getPackInfo(langCode)
+		pack, err := getPacksInfoLang(langCode)
 		if err != nil {
-			return echo.NewHTTPError(http.StatusBadRequest, err.Error())
+			statusCode := http.StatusBadRequest
```

Comment:
does status code matters  to end user ?
Log error

Reply:
Yes, it's a better REST practice, to identify error by status code rather than checking the response text.